lib/WsdlToPostmanCollectionMapper.js

```diff
@@ -339,7 +339,7 @@ class WsdlToPostmanCollectionMapper {
   createItemsFromOperations(operations, urlVariablesData, securityPolicyArray, xmlParser) {
     let items = operations.map((operation) => {
       let requestBody = this.wsdlToPostmanCollectionBodyMapper
-          .getBody(operation, operation.input[0], securityPolicyArray, xmlParser),
+          .getBody(operation, operation.input[0], [operation.header, securityPolicyArray], xmlParser),
```

Comment:
why not add separate arg for `operation.header` instead using array?

Reply:
I would have had to change every function in between `wsdlPostmanCollectionMapper.getBody` to `SOAPMessageHelper.convertInputToMessage`, so I decided to make it into an array as it is only processed at the end at `SOAPMessageHelper.convertInputToMessage`.

Reply:
@aman-v-singh This is not advisable practice. i.e. We should not be changing the argument type whenever required and especially not to array. Let's either use a separate argument for this or use Object as org where you can pass multiple props for better future extensibility.

e.g. something like
```
getBody(operation, operation.input[0], { headers: operation.header, securityPolicyArray }, xmlParser)
```